### ISSUE TITLE
New Desktop Wallet for TRON

### ISSUE THREAD

**Issue by serderovsh** [tronprotocol/tron-web#138]
Hello, today released the first version for wallet for **TRON**
Source code do not judge strictly, the first version. There will be a lot of updates and opportunities accordingly.

- Angular 7 (typescript)
- TronWeb

P.S: If anyone needs a class to support "TronWeb" on "Typescript"
https://github.com/iexbase/iex-wallet/blob/master/src/providers/tron/typings/tron.types.ts

## Main Features

- TRON coin support
- Multiple wallet creation and management in-app
- Easy spending proposal flow for shared wallets and group payments
- Device-based security: all private keys are stored locally, not in the cloud
- Support for TRON testnet wallets
- Support for over 50 currency pricing options and unit denomination in TRX
- Mnemonic (BIP39) support for wallet backups
- Customizable wallet naming and background colors
- Current market value, along with weekly movements.
- Multiple languages supported
- Support sign and verify message
- Secure login with limited login attempts
- Freeze and thaw the balance of different types (bandwidth, energy)

Screenshots: https://github.com/iexbase/iex-wallet/tree/master/screenshots

In a week I will close the topic.

**Comment by serderovsh** [comment]
@Kondax, I did not advertise.
Some wanted support for "TronWeb" for "Typescript"

**Comment by Kondax** [comment]
Yeah, whatever. The issue is literally called "New desktop wallet for TRON". Don't do this again.
